types/react-navigation/index.d.ts

```diff
@@ -268,6 +268,7 @@ export interface NavigationInitAction {
 export interface NavigationResetAction {
   type?: 'Navigation/RESET',
   index: number,
+  key?: string | null,
```

Comment:
ummm...
In that case,  I usually do like this.
```
 reset(undefined)
```
I can't judge now.

Maybe we need to see other typings how to do in this case such as [@types/react-native](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-native).

Keep Same.